.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
   "editor.tabSize": 2,
+  "eslint.enable": false,
```

Comment:
Eslint is cool, it makes the code nicer and saves you some bugs. I suggest that you use it in your text editor.
Anyway, pushing this settings folder on the repo was a mistake, I removed it from master.